testing/oak_echo_raw_bin/build.rs

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+fn main() {
+    println!("cargo:rerun-if-changed=layout.ld");
+    println!("cargo:rustc-link-arg=--script=layout.ld");
```

Comment:
@andrisaar @conradgrobler @ipetr0v for the time being, can we at least move this linker script to the root of the workspace, and use it from all the crates, via `env!("WORKSPACE_ROOT")`? We have too many copies already, it's getting out of hand.

Reply:
This should be a separate PR, but I agree, we need to try creating a common crate that will be shared across different TEE binaries.